setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
We are using RTD successfully along with Cython in `PyWavelets`.  I believe @rgommers was the one who set it up, so I don't have direct experience with it, but from what I can tell it looks like we have configured RTD to use a `pip requirements file` under the "Advanced Settings" tab of the the admin section on readthedocs.org.  This file points to the following requirements.txt that specifies that Cython should be installed:

https://github.com/PyWavelets/pywt/blob/master/util/readthedocs/requirements.txt